docs/include/conf/cincinnati/visiting.rst

```diff
@@ -0,0 +1,69 @@
+Visiting Cincinnati
+===================
+
+Wondering where to stay and how to get around once you get into Cincinnati?
+
+The conference will be held at `Union Hall <http://unionhallcincy.com>`_ in the heart of Cincinnati's historic `Over-the-Rhine <https://en.wikipedia.org/wiki/Over-the-Rhine>`_ neighborhood.
+
+Eating
+------
+
+Snacks and drinks are provided throughout the conference. For meals during the conference, connect with new friends and explore one of Cincinnati's most diverse neighborhoods. Here are some resources that will help you find something good to eat:
+
+- The `Over-the-Rhine Chamber of Commerce <http://business.otrchamber.com/list/ql/restaurants-food-beverages-22?_ga=2.247727175.1927584494.1524751496-300914214.1524751496>`_ lists a number of restaurants and bars in the neighborhood
+- `Open Table <https://www.opentable.com/n/cincinnati/over-the-rhine-restaurants>`_ also lists great restaurants in Over-the-Rhine
+
+Where to stay
+-------------
+
+We are not working with any of the hotels for attendee accommodations, but there are many hotels available within walking distance of the venue:
+
+- `Symphony Hotel <http://www.symphonyhotel.com>`_
+- `The Cincinnatian Hotel <http://www.cincinnatianhotel.com>`_
+- `21c Museum Hotel <https://www.21cmuseumhotels.com/cincinnati/>`_
+- `Millennium Cincinnati <https://www.millenniumhotels.com/en/cincinnati/millennium-hotel-cincinnati/>`_
+- `Hilton Cincinnati Netherland Plaza <http://www3.hilton.com/en/hotels/ohio/hilton-cincinnati-netherland-plaza-CVGNPHF/index.html>`_
```

Comment:
Most of these are pretty pricey. Most affordable downtown hotels are probably Millenium and the new Holiday Inn. I've also directed people at the hotels in Newport if they're on a tight budget, and told them about the Southbank Shuttle.

Reply:
Thank you, Shaun! I added Holiday Inn, shuffled the list a bit, and added a note about hotels in Newport and the Southbank Shuttle.